doc/other_modules/jenny/yarn_project.md

```diff
@@ -0,0 +1,16 @@
+# Yarn Project
+
+## Creating YarnProject
+
+- **variableStorage**
+- ?
+
+
+## Adding user-defined functions
+
+TODO
+
```

Comment:
Should this be added in this PR, or later?

Reply:
Since the PR is getting too big already, it's probably better to leave some topics for later.

Reply:
Sounds good